modules/calendar/add.php

```diff
@@ -0,0 +1,548 @@
+    <!--Add Vacation tab-->
+    <form class="form-horizontal vacation-form" id="provider_form" action="add.php" method="POST" style="margin-top: 20px;">
+      <div class="form-group">
+        <label class="control-label col-sm-2" for="startDate">Start date</label>
+        <div class="col-sm-4">
+          <input type="text" class="form-control" id="startDate" name="startDate" placeholder="Date">
+        </div>
+        <label class="control-label col-sm-2" for="endDate">End date</label>
+        <div class="col-sm-4">
+          <input type="text" class="form-control" id="endDate" name="endDate" placeholder="Date">
+        </div>
+      </div>
+      <div class="form-group">
+        <label class="control-label col-sm-2" for="provider_id">Provider</label>
+        <div class="col-sm-10">
+          <select class="form-control" id="provider_id" name="provider_id">
+            <option value="0">Choose Provider</option>
+            <?php
+            $providers = getProviderInfo();
+            foreach($providers as $provider) {
+              $id = $provider['id'];
+              $selected = ($_POST['provider_id'] == $id) ? "selected" : "";
+              echo "<option value='$id' $selected>" . htmlspecialchars($provider['lname'],ENT_QUOTES) . ", " . htmlspecialchars($provider['fname'],ENT_QUOTES) . "</option>\n";
+            }
+            ?>
+          </select>
+        </div>
+      </div>
+      <div class="form-group">
+        <div class="col-sm-offset-2 col-sm-10">
+          <input type="hidden" name="form_action" value="vacation_submit">
+          <button type="submit" class="btn btn-default" id="vacation_submit">Submit</button>
+        </div>
+      </div>
+    </form>
+    <!--Add Vacation tab-->
+
+
+    <!--Add Holiday tab-->
+    <form class="form-horizontal holiday-form" id="clinic_form" action="add.php" method="POST" style="margin-top: 20px;">
+      <div class="form-group">
+        <label class="control-label col-sm-2" for="addDate">Date</label>
+        <div class="col-sm-4">
+          <input type="text" class="form-control" id="addDate" name="addDate" placeholder="Date">
+        </div>
+      </div>
+      <div class="form-group">
+        <div class="col-sm-offset-2 col-sm-4">
+          <input type="radio" id="radio-choice-1" name="holiday-actions" value="Add Holiday" class="holiday-action">
+          <label for="radio-choice-1">Add Holiday(s)</label>
+        </div>
+        <div class="col-sm-4">
+          <input type="radio" id="radio-choice-2" name="holiday-actions" value="Delete Holiday" class="holiday-action">
+          <label for="radio-choice-2">Delete Holiday(s)</label>
+        </div>
+      </div>
+      <div class="form-group">
+        <label class="control-label col-sm-2" for="clinic_id">Clinic</label>
+        <div class="col-sm-10">
+          <select class="form-control" id="clinic_id" name="clinic_id">
+            <option value="0" data-text="Undefined">Choose Clinic/Facility</option>
```

Comment:
An option for "All" facilities would be nice to have.

Reply:
@teryhill So, if user makes a holiday with a date, day type and facility option "All", what should be stored in DB in facility id? I mean how should it work?

Reply:
The facilities listed in the facility table. You could wait on this also and see if the customers want it.